trackpy/motion.py

```diff
@@ -154,13 +154,14 @@ def compute_drift(traj, smoothing=0, pos_columns=['x', 'y']):
     >>> corrected_traj = subtract_drift(traj, drift) # Apply them.
     """
     # Probe by particle, take the difference between frames.
-    delta = pd.concat([t.set_index('frame', drop=False).diff()
-                       for p, t in traj.groupby('particle')])
+    delta = traj.groupby('particle').apply(lambda x :
```

Comment:
minor comment, we follow pep8 so can you put spaces after the commas?  It seems silly, but having a consistent style really does help in code bases maintained by a group of people.